src/java.base/share/classes/java/lang/reflect/Executable.java

```diff
@@ -51,21 +51,44 @@
  */
 public abstract sealed class Executable extends AccessibleObject
     implements Member, GenericDeclaration permits Constructor, Method {
+    // fields injected by hotspot
```

Comment:
If a field is listed here then it is NOT injected by hotspot.

Reply:
What would be the terminology for a final field that's set by hotspot, against the regular java constrcutor rules?

Reply:
I have chosen the wording "all final fields are used by the VM"

Reply:
I don't know of any specific terminology - we typically just add a comment saying the field is set and/or read by the VM.